src/common/number/format_number.ts

```diff
@@ -82,17 +82,14 @@ export const formatNumber = (
     !Number.isNaN(Number(num)) &&
     num !== "" &&
     localeOptions?.number_format === NumberFormat.none &&
-    Intl &&
-    (options?.maximumFractionDigits != null ||
-      options?.minimumFractionDigits != null)
+    Intl
   ) {
-    // If NumberFormat is none, just set the digits options for precision and use en-US format without grouping.
+    // If NumberFormat is none, use en-US format without grouping.
```

Comment:
Always use this function so any format will be used for format none (except grouping)